File: engine/model/src/main/scala/tools/packet/item/enhance/SendVegaScroll.scala
package tools.packet.item.enhance

import net.opcodes.SendOpcode
import server.MaplePacketOpCodes
import tools.packet.PacketInput

class SendVegaScroll(private var _operation: MaplePacketOpCodes.VegaScroll) extends PacketInput {
  def operation: MaplePacketOpCodes.VegaScroll = _operation

  override def opcode(): SendOpcode = SendOpcode.VEGA_SCROLL
}

File: engine/engine-model/src/main/java/tools/packet/item/enhance/SendVegaScroll.java
package tools.packet.item.enhance;

import net.opcodes.SendOpcode;
import server.MaplePacketOpCodes;
import tools.packet.PacketInput;

public record SendVegaScroll(MaplePacketOpCodes.VegaScroll operation) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.VEGA_SCROLL;
   }
}